recipes/aionotify/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "aionotify" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "64b702ad0eb115034533f9f62730a9253b79f5ff0fbf3d100c392124cdf12507" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: "{{ sha256 }}"
+
+build:
+  number: 0
+  skip: True  # [not linux]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - python >=3.4
+  run:
+    - python >=3.4
+
+test:
+  imports:
+    - aionotify
+  requires:
+    - asynctest
```

Comment:
Since you're not running the actual test suite, `asynctest` shouldn't be needed

Reply:
the changes have been made